lib/rust/ensogl/component/text/src/component/area.rs

```diff
@@ -824,7 +822,14 @@ impl AreaModel {
             let next = iter.next();
             let style = line_style.next().unwrap_or_default();
             let chr_size = style.size.raw;
-            let char_info = next.as_ref().map(|t| pen::CharInfo::new(t.1, chr_size));
+            let char_info = next.as_ref().map(|t| {
```

Comment:
Lets make it `map(|(<first field>, <second field>)|` with meaningful field names, so we avoid using .0 and .1 here.